config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -393,7 +394,7 @@ presubmits:
         - --flush-mem-after-build=true
         - --gcp-node-size=e2-standard-8
         - --gcp-nodes=84
-        - --gcp-project=k8s-presubmit-scale
+        - --gcp-project-type=scalability-project
```

Comment:
Can scalability-project handle 5k-node kubemark?

Don't we want a different class for those projects?

Reply:
yes, scalability-project can handle 5k kubemarks (our kubemark 5k CI already uses this pool). 
We don't have different class right now.